src/Costura/CallAttach.cs

```diff
@@ -4,12 +4,18 @@
 
 partial class ModuleWeaver
 {
-    void CallAttach()
+    void CallAttach(Configuration config)
     {
-        if (FindInitializeCalls())
-            return;
+        var initialized = FindInitializeCalls();
 
-        AddModuleInitializerCall();
+        if (config.LoadAtModuleInit)
```

Comment:
I just realized on Windows you could just use CosturaUtility.Initializer anyway, but whatever. It's safe to call multiple times.